client/src/components/myBookings/myBookings.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import ParkingSpot from '../shared/parkingSpot/parkingSpot.jsx';
+import Trash from '../shared/icons/trash.jsx';
+import MyBookingsHeader from './myBookingsHeader.jsx';
+import './myBookings.css';
+
+class MyBookings extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bookings: []
+    };
+  }
+
+  onClick = () => {
+    console.log('CLICKED DELETED')
+  }
+
+  componentDidMount () {
+    fetch(`/my-bookings/${this.props.userId}`)
+      .then((res) => res.json())
+      .then((bookings) => {
+        console.log('MY BOOKINGS', bookings);
+        let requiredBookings = bookings.map((booking) => {
+          let duration = (Number(booking.end_time) - Number(booking.start_time)) / 3600;
+          let total = duration * booking.price;
+
+          let unixStart = booking.start_time;
+          let unixEnd = booking.end_time;
+          let humanStartTime = new Date(unixStart * 1000);
```

Comment:
Do you think it makes more sense to just store the whole UNIX timestamp in the db? It was a suggestion that I remove milliseconds before saving the timestamp but I realize now that you actually need them to get the new Date object to return the correct time. Let me know and I can change the code in my component back and you won't have to multiply by 1000. Maybe it doesn't really matter, though.